### PR TITLE
Fix ring size to 3 from v8

### PR DESCRIPTION
**Pull request by stoffu**

### CODE REVIEW

File: src/simplewallet/simplewallet.cpp [hunk 1]
```diff
@@ -2389,7 +2394,7 @@ simple_wallet::simple_wallet()
                                   "store-tx-info <1|0>\n "
                                   "  Whether to store outgoing tx info (destination address, payment ID, tx secret key) for future reference.\n "
                                   "default-ring-size <n>\n "
-                                  "  Set the default ring size (default and minimum is 5).\n "
+                                  "  Set the default ring size (default and minimum is 3).\n "
```
Comment (iamsmooth) [inline src/simplewallet/simplewallet.cpp]: Should be marked as obsolete here too?

Comment (stoffu) [reply src/simplewallet/simplewallet.cpp]: It's addressed by the second commit.

**Comment by stoffu** [comment]
For those who already ran the testnet daemon on v8 (k12) fork, the existing blocks became incompatible with the change proposed here at around 112380, so either pop enough number of blocks using `aeon-blockchain-import`, or simply sync the chain from scratch again.

**Comment by stoffu** [comment]
Includes https://github.com/monero-project/monero/pull/5874

**Pull request merged**